dotnet/src/webdriver/WebDriverError.cs

```diff
@@ -218,14 +213,23 @@ static WebDriverError()
             resultMap[ScriptTimeout] = WebDriverResult.AsyncScriptTimeout;
             resultMap[SessionNotCreated] = WebDriverResult.SessionNotCreated;
             resultMap[StaleElementReference] = WebDriverResult.ObsoleteElement;
+            resultMap[DetachedShadowRoot] = WebDriverResult.DetachedShadowRoot;
             resultMap[Timeout] = WebDriverResult.Timeout;
             resultMap[UnableToSetCookie] = WebDriverResult.UnableToSetCookie;
             resultMap[UnableToCaptureScreen] = WebDriverResult.UnableToCaptureScreen;
             resultMap[UnexpectedAlertOpen] = WebDriverResult.UnexpectedAlertOpen;
             resultMap[UnknownCommand] = WebDriverResult.UnknownCommand;
-            resultMap[UnknownError] = WebDriverResult.UnhandledError;
-            resultMap[UnknownMethod] = WebDriverResult.UnknownCommand;
-            resultMap[UnsupportedOperation] = WebDriverResult.UnhandledError;
+            resultMap[UnknownError] = WebDriverResult.UnknownError;
+            resultMap[UnknownMethod] = WebDriverResult.UnknownMethod;
+            resultMap[UnsupportedOperation] = WebDriverResult.UnsupportedOperation;
+
+            // TODO: Remove these strings; it is no longer valid in the specification.
```

Comment:
Just remove, it is internal implementation.

Reply:
Deleted and obsoleted public-facing error string

Reply:
`WebDriverError` is `internal` so I can remove the strings. Removed.